src/middlewared/middlewared/plugins/jail_freebsd.py

```diff
@@ -715,9 +715,10 @@ def default_configuration(self):
         Retrieve default configuration for iocage jails.
         """
         if not self.iocage_set_up():
-            return IOCJson.retrieve_default_props()
+            defaults = IOCJson.retrieve_default_props()
         else:
-            return self.query(filters=[['host_hostuuid', '=', 'default']], options={'get': True})
+            defaults = self.query(filters=[['host_hostuuid', '=', 'default']], options={'get': True})
+        return {k: v for k, v in defaults.items() if k not in IOCJson.default_only_props}
```

Comment:
is the `if k not in IOCJson.default_only_props`, feels odd at first glance, is it not the opposite?